docs/SMARTonFHIR.md

```diff
@@ -0,0 +1,44 @@
+[SMART on FHIR](https://docs.smarthealthit.org/) is a set of open specifications to integrate partner applications with FHIR servers and electronic medical records systems that have FHIR interfaces. 
+
+There is a tutorial on the managed service documentation that describes how to use the proxy to enable SMART on FHIR applications with the Azure API for FHIR. You can follow the [steps](https://docs.microsoft.com/azure/healthcare-apis/use-smart-on-fhir-proxy) for the open-source FHIR Server as well with a few changes that are highlighed below.
+
+The first step is to make updates/validate settings for your app service. Navigate to the App Service that was greated as part of the deployment,
+
+# Enable the SMART on FHIR proxy
+1. Select **Configuration** under Settings
+1. Set FhirServer:Security:EnableAadSmartOnFhirProxy and FhirServer:Security:Enabled to **True**.
+
+![App Service Settings](images/SMARTonFHIR/app-service-settings.png)
+
+The OSS FHIR Server security is handled through the server (or app) using four custom defined [roles](https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.Shared.Web/roles.json), globalAdmin, globalWriter, globalReader and globalExporter. It is not done through the portal, and therefore you can leave the App Service Automation in the default setting, "Off".
```

Comment:
>The OSS FHIR Server security is handled through the server (or app) using four custom defined [roles](https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.Shared.Web/roles.json), globalAdmin, globalWriter, globalReader and globalExporter [](start = 0, length = 263)

I'm not sure what is purpose of this sentence. Only time roles mentioned is in `Troubleshoot the App` section. Roles discussed in tutorials for Resource and Client application registering, but they have good coverage in them already.

Reply:
Agreed. i'm going to remove the whole section.